### PR TITLE
Created welcome controller action and view to test heroku

### PR DESCRIPTION
**Pull request by EricLarson2020**
Created welcome page to test the functionality of heroku application

### CODE REVIEW

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class WelcomeController < ApplicationController
```
Comment (EricLarson2020) [inline app/controllers/welcome_controller.rb:1]: This is to test functionality of heroku, will change when building out welcome page

Comment (stellakunzang) [reply app/controllers/welcome_controller.rb:1]: this is great since Jack and I need it to in order to test the nav bar and display on local host!

Comment (sagemlee) [reply app/controllers/welcome_controller.rb:1]: Great thinking Eric!

**Pull request merged**